camelot/admin/action/list_filter.py

```diff
@@ -199,13 +199,15 @@ def assert_valid_attribute(cls, attribute):
         assert issubclass(attribute.type.python_type, cls.python_type), 'The python_type of the given attribute does not match the python_type of this search strategy'
     
     @classmethod
-    def get_clause(cls, attribute, text, field_attributes):
+    def get_clause(cls, search_strategy, text, field_attributes, attribute=None):

```

Comment:
is er een reden waarom het een classmethod blijft ? als je de instance toch mt meegeven ?

Reply:
Ja, opdat de methode kan aangesproken worden op class objecten (search_strategy attribute in de field_attributes) en op instance objecten (list_search). Eigenlijk zouden dat allemaal instances moeten zijn, maar in de field_attributes kent ge het attribuut nog niet altijd op het moment van constructie. Ik ben al aan het denken geweest om die later te initialiseren zoals dat ook met de delegates gebeurt.